app/models/user.rb

```diff
@@ -3,4 +3,11 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
   enum gender: { male: 0, female: 1 }
+  accepts_nested_attributes_for :interests, reject_if: :all_blank, allow_destroy: true
+
+  private
+  def self.create_with_password(attr={})
+    default_password = "secret"
```

Comment:
You can just use ```"secret``` as a password in the line below. It is not used many times so there is no need to extract it. 
Consider moving this variable to const.